src/shared/components/oncoprint/SortUtils.ts

```diff
@@ -302,6 +303,32 @@ export function getClinicalTrackSortComparator(track: ClinicalTrackSpec) {
     };
 }
 
+export function getClinicalTrackInitialSortDirection(track: ClinicalTrackSpec) {
+    let initialSortOrder: 0 | 1 | -1 | undefined = 0 as 0;
```

Comment:
```suggestion
    let initialSortOrder = 0;
```

Reply:
If I do this I get type 'number' is not assignable to type 0 | 1 | -1. So I need to specify the type of initialSortOrder by either writing 
let initialSortOrder = 0 as 0 | 1 | -1;
or
let initialSortOrder : 0 | 1 | -1  = 0;

Reply:
Ok, I see.